Asterix/tests/test_coronagraph.py

```diff
@@ -26,6 +26,7 @@ def test_all_coronagraphs():
         # Create the coronagraph
         corono = Coronagraph(modelconfig, Coronaconfig)
         coro_psf = corono.todetector_intensity(in_contrast=True)
+        # print(f"Attenuation of '{coro} is { np.max(coro_psf)}")
```

Comment:
Same like comment above:
Please delete. Commented print statements are very bad coding. If you need it in the future you can add it temporarily, otherwise it just clutters the code.
Especially since print statements are ignored in test functions.

Reply:
I've uncommented it. This is fairly useful to check all coronagraphs best performance and do not appears anyway in test mode only when we run them in the main

Reply:
Sounds good! Also, I have found this, which clarifies how pytest handles stoud, it actually will print it if the test fails:
https://stackoverflow.com/a/24617813/10112569